doc/api/diagnostics_channel.md

```diff
@@ -977,6 +977,44 @@ channels.asyncStart.bindStore(myStore, (data) => {
 });
 ```
 
+#### `tracingChannel.hasSubscribers`
+
+<!-- YAML
+added:
+ - v22.0.0
+ - v20.13.0
+-->
+
+> Stability: 1 - Experimental
+
+* Returns: {Boolean} `true` if any of the individual channels has a subscriber,
+  `false` if not.
+
+This is a helper method available on a [`TracingChannel`][] instance to check if
```

Comment:
Maybe something like:
```md
This is a helper method available on the [`TracingChannel`][] instance to check whether any [TracingChannel Channels][] have subscribers. `true` is returned if any of them have at least one subscriber, `false` otherwise.
```

Reply:
Just curious, why the 👎?

I believe the wording I suggested better conveyed that "subscribers" means "at least one" (if for some reason someone didn't understand that)

I also think the wording I suggested better conveyed the true/false returns in less wording.

If you disagree, I'd love to know why :-) it's your PR, so you have the final say.